tests/functional/api/BackbeatServer.js

```diff
@@ -263,7 +263,20 @@ describe('Backbeat Server', () => {
             statsClient.reportNewRequest(`${site2}:${OPS_FAIL}`, 55);
             statsClient.reportNewRequest(`${site2}:${BYTES_DONE}`, 1874);
             statsClient.reportNewRequest(`${site2}:${BYTES_FAIL}`, 575);
+
+            // TODO: quick fix made for failures/pending
```

Comment:
? is a quick fix needed or is this an old TODO that has been completed in this commit

Reply:
Yup, I want to keep this todo. I want to improve these tests, but currently not sure how atm

Reply:
Can you update the TODO to be more descriptive of future work? maybe something like 'refactor/improve tests for failures/pending, current tests are modified as a quick fix'. Sorry for being so picky, the current `TODO` sounds more like 'here's what happened' than a 'this is what should happen'